extensions/amp-twitter/1.0/storybook/Basic.js

```diff
@@ -25,7 +25,57 @@ export default {
 };
 
 export const _default = () => {
+  const tweetId = select(
+    'tweet id',
+    ['1356304203044499462', '495719809695621121', '463440424141459456'],
+    '1356304203044499462'
+  );
+  const cards = boolean('show cards', true) ? undefined : 'hidden';
+  const conversation = boolean('show conversation', false) ? undefined : 'none';
+  return (
+    <Twitter
+      bootstrap="https://3p.ampproject.net/2104170104001/f.js"
```

Comment:
I'd like to understand more about passing `bootstrap` and `src`. It seems like it's very specific to AMP's concept of 3p frames. How would this be used in a Preact context?

Reply:
Good question! Ideally these will have default values so that they don't have to be provided and the component will "just work" -- however, that requires more file reorganization and isolation from AMP than the default behavior currently has. Until then, and when at that future point we do have defaults that are (potentially) tied to an AMP RTV, this is an example of how a publisher could provide alternative `bootstrap` or `src` values to opt out of using logic served by AMP. 

I agree this isn't a great example of what publishers would use instead here, as it might look like recreating some of what we have in 3p/twitter.js and hosting it somewhere else (like on the publisher's own domain).

Reply:
In a preact context, would a 3p iframe be used at all?

Reply:
It's not clear, so it would be helpful to have a working component to get feedback from. :)

Reply:
Message received. Approval incoming